fastlane/docs/FAQs.md

```diff
@@ -1,3 +1,117 @@
-<h3 align="center">
```

Comment:
This file/change is not wanted here

Reply:
I am still trying to figure out how to remove the unwanted commits. I merged `master` into my branch and got these changes. 🤷‍♂️

Reply:
You will get better results if you rebase your branch onto master. (Meaning: Moving your new commits from where they were onto the most recent commits on master)

Reply:
That's what I am trying to do. But when I rebase I get multiple merge conflicts. Here is a screen cap.

https://www.dropbox.com/s/vldkuudauduvvmh/Untitled.mov?dl=0

Reply:
What are you clicking after you did "Resovle using theirs"? It is enough to have the changes all staged, then you can "Continue to rebase". I think you might be pushing at this moment again which fails?